src/Components/SuperAdmins/super-admins.module.css

```diff
@@ -1,89 +1,124 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
 .container {
-  flex: 1;
   display: flex;
   justify-content: center;
   align-items: center;
-  background-image: url('../../../public/assets/images/super-admin-background.png');
-  background-size: cover;
-  font-size: 1.5vw;
+  background-image: url('../../../public/assets/images/subscriptions-background.jpg');
 }
 
-.supAdmInput, .supAdmBtn {
-  font-size: 1.3vw;
-}
-
-.box {
-  color: white;
-  width: 85vw;
+.list {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: 50px;
   background-color: rgba(55, 56, 103, 0.5);
-  padding: 2vw;
-  border-radius: 1vw;
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  font-family: 'Roboto', sans-serif;
 }
 
-.table{
+.header {
   display: flex;
-  width: 95%;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.title {
+  color: white;
+  font-size: 40px;
+  line-height: 48px;
 }
 
-.supAdmTr:nth-child(even) {
-  background-color: rgba(55, 56, 103, 0.7);
+.btn {
+  width: 150px;
+  height: 50px;
+  background-color: #46A37C;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 19px;
+  color: white;
+  text-transform: uppercase;
+  cursor: pointer;
 }
 
-.supAdmTh {
+.table {
   text-align: left;
+  align-self: center;
+  width: 80%;
+  border-collapse: collapse;
 }
 
-.tdText {
-  width: 40vw;
+.thead {
+  padding: 8px;
 }
 
-.tdBtn {
-  text-align: center;
-  width: 12vw;
+.tr {
+  color: white;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
 }
- 
-.titleForm, .titleTable{
-  text-align: center;
+
+.td {
+  border-bottom: 1px solid #FFFFFF;
+  color: white;
+  font-size: 20px;
+  line-height: 24px;
 }
 
-.formPost {
-  display:flex;
-  justify-content: space-around;
-  align-items: center;
-  height: 5vw;
+.form {
+  margin-top: 50px;
+  background-color: rgba(55, 56, 103, 0.05);
+  font-weight: 800;
+  font-size: 20px;
+  line-height: 24px;
+  color: #FFFFFF;
 }
 
-.submitBtn {
-  width: 10vw;
-  height: 4vw;
-  color: white;
-  font-weight: bold;
-  background-color: green;
-  border:none; 
-  border-radius: 0.5vw;
-  cursor:pointer;
+.label {
+  padding-right: 20px;
+  width: 10%;
+  margin-left: 10%;
 }
 
-.acceptBtn {
-  width: 7vw;
-  height: 2vw;
-  color: white;
-  font-weight: bold;
-  background-color: green;
-  border:none; 
-  border-radius: 0.5vw;
-  cursor:pointer;
+.inputBox {
+  display: flex;
+  align-items: center;
 }
 
-.cancelBtn {
-  width: 7vw;
-  height: 2vw;
+.btnAdd {
+  margin-top: 10px;
+  margin-left: 10%;
+  width: 150px;
+  height: 60px;
+  background-color: #46A37C;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 19px;
   color: white;
+  text-transform: uppercase;
+  cursor: pointer;
+}
+
+.btnEdit {
```

Comment:
Same here

Reply:
They were left over from the original css and I forgot to remove them. They are currently not used.